internal/utils/utils.go

```diff
@@ -22,22 +26,29 @@ func RemoveUnusedTempDir() {
 
 	for _, f := range files {
 		if f.IsDir() && strings.HasPrefix(f.Name(), "vxlua-") {
-			pathToPID := filepath.Join(os.TempDir(), f.Name(), "lock.pid")
-			fdata, err := ioutil.ReadFile(pathToPID)
+			fdata, err := ioutil.ReadFile(filepath.Join(os.TempDir(), f.Name(), "lock.pid"))
+			path := filepath.Join(os.TempDir(), f.Name())
 			if err != nil {
-				os.RemoveAll(filepath.Join(os.TempDir(), f.Name()))
+				removeAll(path)
 				continue
 			}
 			pid, err := strconv.Atoi(string(fdata))
 			if err != nil {
-				os.RemoveAll(filepath.Join(os.TempDir(), f.Name()))
+				removeAll(path)
 				continue
 			}
 			proc, _ := os.FindProcess(pid)
 			if proc == nil || err != nil {
-				os.RemoveAll(filepath.Join(os.TempDir(), f.Name()))
+				removeAll(path)
 				continue
 			}
 		}
 	}
 }
+
+func removeAll(path string) {
+	e := os.RemoveAll(path)
```

Comment:
I wonder why this is not working https://github.com/vxcontrol/soldr/blob/master/.golangci.yml#L77

This should be ignored by linter

Reply:
In this case, the linter ignores this line. However, it is strange that in the original version he did not do this.

Reply:
I put the duplicate code into the function. It was repeated three times in one function.